index.yaml

```diff
@@ -11459,35 +11414,6 @@ entries:
     urls:
     - assets/rancher-logging/rancher-logging-102.0.2+up3.17.10.tgz
     version: 102.0.2+up3.17.10
-  - annotations:
```

Comment:
@vardhaman22 why is this PR removing this `rancher-logging` entry on `index.yaml`?

Reply:
updated..theses changes are no longer present ..fixed by https://github.com/rancher/charts/pull/3613